src/PharoThings-Devices-LCD1602.package/PotLCD1602Device.class/instance/backlight..st

```diff
@@ -0,0 +1,10 @@
+controlling
+backlight: anValue
+	|brigthness|
+	brigthness := anValue.
+	anValue > 1024 ifTrue: [ brigthness := 1024 ].
+	anValue < 0 ifTrue: [ brigthness := 0 ].
+	backlight value: 1024 - brigthness.
```

Comment:
is backlight is a pin instance? Then it would be more clear and consistent to name it backlightPin

Reply:
This is from the previous package. Can be removed also.